http/metrics.go

```diff
@@ -52,11 +52,16 @@ var (
 		},
 		[]string{"code", "method", "source"},
 	)
+
+	paceHTTPPanicCounter = prometheus.NewGauge(prometheus.GaugeOpts{
```

Comment:
could you add the new metric to the readme?

Reply:
added reference to the metric in the `metrics/README.md`, but it feels kinds of out-of-place. If there is a better place to put it, let me know